arn/validate/Validate.go

```diff
@@ -55,6 +58,16 @@ func Date(date string) bool {
 	return err == nil
 }
 
+// YearMonth tells you whether the date contain only the year and the month.
```

Comment:
"contains".

Please run the [golangci-lint](https://github.com/golangci/golangci-lint) linter before submitting because it includes a lot of good linters including [misspell](https://github.com/client9/misspell). Our repository includes a config file for golangci-lint.